src/components/guideLineLayer.ts

```diff
@@ -89,6 +89,14 @@ export module Components {
       }
     }
 
+    protected _setPixelPositionWithoutChangingMode(pixelPosition: number) {
```

Comment:
If I just called `pixelPosition()` on dragging, that would change the mode, meaning the line wouldn't keep the same `value()` as the scale updated.